### DISCUSSION TITLE
Migrate from FCOS to SCOS

### DISCUSSION THREAD

**Discussion by forzamehlano** [okd-project/okd#1860] [General]
As per the title, is it possible to migrate a cluster built on FCOS to SCOS? We've been running on FCOS since 4.5 and have run into some bugs on the 6.x kernel of FCOS that have us looking at potential options to mitigate them.

One of them is using SCOS instead which aligns closer to RHCOS than FCOS but is there a migration path / documentation on how this might be achieved?

Any other info/experiences on running OKD with SCOS in production? Is it ready etc etc?

Thanks
Nick

**Comment by vrutkovs** [comment]
You force upgrade from OKD FCOS to OKD SCOS, yes, via `oc adm release upgrade --to-image`

**Reply by forzamehlano** [reply]
Thanks @vrutkovs.

Are there any plans to test SCOS upgrades in the same way as the FCOS releases (or if the testing already takes place to include them in the upgrade graph)? 

Having a *tested* upgrade path between releases feels quite important.

**Reply by vrutkovs** [reply]
That's a question for https://github.com/okd-project/okd-scos tbh
